en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-configuration.md

```diff
@@ -0,0 +1,30 @@
+# Setting up ISO8583 Connector 
+
+ISO8583 is an international standard for financial transaction card originated interchange messaging. It is the International Organization for Standardization standard for systems that exchange electronic transactions initiated by cardholders using payment cards.
```

Comment:
"ISO8583 is an international standard for financial transaction card originated interchange messaging." Can we simplify this? I see some similar text in wikipedia and it's a bit too complex. Can we simplify this definition?

Reply:
Updated.